docker/docker-compose.kafka-setup.yml

```diff
@@ -1,15 +1,4 @@
-# Service definitions for Kafka Setup container.
+# Empty docker compose for kafka-setup as we have moved kafka-setup back into the main compose
```

Comment:
Can this just be removed? It's not defining any services

Reply:
The CLIs that were installed in the last couple of days will try to pull this file, this way they won't fail.